src/implementations/vanilla/components/carousel/_carousel.scss

```diff
@@ -225,6 +264,37 @@ $_outline-width: null !default;
 
   .ecl-carousel__prev,
   .ecl-carousel__next {
-    display: block;
+    padding: map.get(theme.$spacing, 's');
+    position: absolute;
+    top: calc(50% - 22px);
+    transform: translateY(-50%);
+    z-index: 1;
+  }
+
+  .ecl-carousel__prev {
+    left: 0;
```

Comment:
I guess that it should be inverted in the rtl css

Reply:
Already done in RTL CSS resource